nvmeadm/src/nvmf_subsystem.rs

```diff
@@ -243,15 +243,16 @@ impl Subsystem {
 
     /// Returns the particular subsystem based on the nqn and address.
     // TODO: Optimize this code.
-    pub fn get(host: &str, port: &u16, nqn: &str) -> Result<Subsystem, NvmeError> {
+    pub fn get(host: &str, port: &u16, transport: &str, nqn: &str) -> Result<Subsystem, NvmeError> {
```

Comment:
use an enum here

Reply:
> use an enum here

and then convert to `&str` during comparison with `subsys.transport` ?

Reply:
Yes that's it

Reply:
It's already done in latest diff, PTAL.